app/controllers/main_controller.rb

```diff
@@ -1,5 +1,12 @@
 # frozen_string_literal: true
 
 class MainController < ApplicationController
-  def index; end
+  def index
+    # @user = User.find(params[:id])
+  end
+
+  def edit
+    
+  end
+  
 end
```

Comment:
¿?

Reply:
forget undo that unnecessary changes, sorry!